<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logger</class>
 <widget class="QDialog" name="Logger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Viewer:</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="3" column="0" colspan="5">
       <widget class="QTableView" name="logView">
        <property name="minimumSize">
         <size>
          <width>470</width>
          <height>129</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>35</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>19</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="iconViewer">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="autoScroll">
        <property name="text">
         <string>Auto-Scroll</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="clearLogsButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Clear Logs</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="filterLabel">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Filter:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filterComboBox">
          <item>
           <property name="text">
            <string>All Logs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Admin Usage Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Comment Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Usage Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UPNP Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Punishment Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Misc Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Chat Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Quest Log</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Packet Forge</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ping Log</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="autoClear">
        <property name="text">
         <string>Auto Clear Logs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QFrame" name="labelFrame">
        <property name="minimumSize">
         <size>
          <width>235</width>
          <height>75</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>ReMix Game Server</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="versionLabel">
           <property name="text">
            <string>Version [ %1 ]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="copyrightLabel">
           <property name="text">
            <string>Copyright [ NA ]</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="websiteLabel">
           <property name="text">
            <string>Website: &lt;a href=&quot;%1&quot;&gt;&lt;span style=&quot; text-decoration: &quot;
                          &quot;underline; color:#007af4;&quot;&gt;https://bitbucket.org/ahitb/remix&lt;/span&gt;&lt;/a&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QWidget" name="activityGraph" native="true">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
